android/features/music-player/src/main/java/io/newm/feature/musicplayer/repository/MusicRepository.kt

```diff
@@ -4,9 +4,9 @@ import android.content.Context
 import android.net.Uri
 import io.newm.feature.musicplayer.models.Playlist
 import io.newm.feature.musicplayer.models.Track
+import io.newm.feature.musicplayer.R
```

Comment:
what's `R`?

Reply:
This is Android specific, it's an import required when referencing resources.